app/Http/Controllers/Auth/ConfirmablePasswordController.php

```diff
@@ -35,6 +35,6 @@ public function store(Request $request): RedirectResponse
 
         $request->session()->put('auth.password_confirmed_at', time());
 
-        return redirect()->intended(route('dashboard', absolute: false));
+        return redirect()->intended(route('employee', absolute: false));
```

Comment:
please revert this to dashboard. we have implemented a dashboard for common purposes not only for employee

Reply:
Reverted all changes done to Dashboard... create a new issue to properly implement the dashboard